### ISSUE TITLE
Undo / redo not working

### ISSUE THREAD

**Issue by eyssette** [antonmedv/codejar#79]
Your app is really great! Unfortunately, _undo / redo_ seems to be broken.
Is there any way to make it work?
Thank you!

**Comment by antonmedv** [comment]
It seems working for me. What is your setup?

**Comment by eyssette** [comment]
I'm using _Codejar_ for a mindmap app [https://mymarkmap.vercel.app/](https://mymarkmap.vercel.app/). _Undo/redo_ doesn't work on Mac OS (Monterey 12.2.1) or Lubuntu (20.04), whatever browser I try to use.

It's the same for the examples on your website [https://medv.io/codejar/](https://medv.io/codejar/).

**Comment by antonmedv** [comment]
Works for me on macOS via Chrome. Also I use USA keyboard.

**Comment by eyssette** [comment]
I'm using a French keyboard.

**Comment by antonmedv** [comment]
Probably this is a reason.

**Comment by eyssette** [comment]
Same problem in a previous version of your app with a solution : https://github.com/antonmedv/codejar/pull/37
But not working for other keyboards : https://github.com/antonmedv/codejar/pull/58

**Comment by eyssette** [comment]
Would that be possible to use keyCode (89 / 90) instead ?

**Comment by eyssette** [comment]
Maybe using keyCode would be better (event if it's supposed to be deprecated). Same comment here : [https://github.com/antonmedv/codejar/issues/73](https://github.com/antonmedv/codejar/issues/73)

**Comment by ShiriCZ** [comment]
I have the same problem. I'm Czech keyboard. The fix is simple just rewrite lines 382 and 385 in isUndo/isRedo function from 
` event.code === 'KeyZ';`
to
`event.keyCode == 90;`
Then it should work on every keyboard layout, because KeyCode is universal.

**Comment by julianpoemp** [comment]
redo & undo still not working (German Keyboard)

**Comment by julianpoemp** [comment]
@antonmedv I just created an PR that solves this issue for everyone who uses other keyboard layouts than QUERTY. Can you please merge it?

**Comment by julianpoemp** [comment]
because this issue is still not fixed I created a repository with a build from my fix. You can install it via

````
npm install --save julianpoemp/codejar-fixed-redo-undo --legacy-peer-deps
````

Repository: https://github.com/julianpoemp/codejar-fixed-redo-undo

**Comment by antonmedv** [comment]
Will try to release it today.

**Comment by antonmedv** [comment]
Released 3.7.0)
